.env.example

```diff
@@ -26,4 +26,5 @@ NORTHSTAR_URI=http://northstar.dev:8000
 NORTHSTAR_API_KEY=abc4324
 
 PHOENIX_URI=https://dev.dosomething.org:8888
+PHOENIX_API_URI=https://staging.beta.dosomething.org/api
```

Comment:
I'd prefer if we actually use Dev url's here
DS-Homestead is set up to use phoenix correctly and there's nothing funky about using those at the same time